run/jobs/test/system.test.js

```diff
@@ -69,19 +72,26 @@ describe('End-to-End Tests', () => {
       `resource.labels.location = "${REGION}" ` +
       `timestamp>="${dateMinutesAgo(new Date(), 5)}"`;
 
-    const entries = await logging.getEntries({
-      filter: preparedFilter,
-      autoPaginate: false,
-      pageSize: 3,
-    });
+    let found = false;
+    for (let i = 0; i < 5; i++) {
```

Comment:
Nit: would it be cleaner to use the `p-retry` module here?

(That should _also_ obviate the need for `delay`.)